src/algos/keyboards.ts

```diff
@@ -125,7 +127,7 @@ export class manager extends AlgoManager {
     'pbtfans',
     'pom plate',
     'QMK',
-    'QWERTY',
+    'QWERTY layout',
```

Comment:
Making this more specific as apparently a lot of people name their pets qwerty or use it as an online handle for some reason.